README.md

```diff
@@ -0,0 +1,38 @@
+[![Lint](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/lint.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/lint.yml)
+[![Tests](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/tests.yml)
+[![Slither Analysis](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/slither.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/slither.yml)
+
+# StaticOracle
+
+StaticOracle is a tool developed under Uniswap's grant program that aims to help developers integrate easily and fast with uniswap's v3 twaps oracles.
+
+This is done by leveraging already existing [WeightedOracleLibrary]() implemented by [Nicolás Chamo](https://github.com/nchamo).
```

Comment:
Where is `WeightedOracleLibrary` pointing to?

Reply:
Fixed, to https://github.com/Uniswap/v3-periphery/pull/146